modules/cli/src/main/scala/scala/cli/launcher/LauncherCli.scala

```diff
@@ -15,23 +19,22 @@ object LauncherCli {
 
   def runAndExit(version: String, options: LauncherOptions, remainingArgs: Seq[String]): Nothing = {
 
-    val logger       = LoggingOptions().logger
-    val cache        = CoursierOptions().coursierCache(logger.coursierLogger(""))
-    val scalaVersion = options.cliScalaVersion.getOrElse(Properties.versionNumberString)
+    val logger        = LoggingOptions().logger
+    val cache         = CoursierOptions().coursierCache(logger.coursierLogger(""))
+    val scalaVersion  = options.cliScalaVersion.getOrElse(Properties.versionNumberString)
+    val snapshotsRepo = Seq(Repositories.central, Repositories.sonatype("snapshots"))
 
-    val scalaCliDependency = Seq(dep"org.virtuslab.scala-cli::cli:$version")
-    val snapshotsRepo = Seq(
-      Repositories.central.root,
-      Repositories.sonatype("snapshots").root
-    )
+    val cliVersion: String =
+      if (version == "latest") resolveLatestScalaCliVersion(cache, snapshotsRepo) else version
```

Comment:
I think `latest` should point to latest stable version and `nightly` should point to latest snapshot. 

TBH I do not know if we need  `latest` so we may just rename it to `nightly`

Reply:
Right, `nightly` sounds better, so I just renamed it.